### PR TITLE
Javadoc

### PR DESCRIPTION
**Pull request by jburel**
fix javadoc warnings
In order to see if some warnings remain, you will need to use Java version > 1.8 (tested with Java 11)
Currently no warnings are returned if Java 1.8 is used
To check run ``gradle javadoc``

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-build-push#28](https://web-proxy.openmicroscopy.org/west-ci/job/OMERO-build-push/28/). See the [console output](https://web-proxy.openmicroscopy.org/west-ci/job/OMERO-build-push/28/consoleText) for more details.
Possible conflicts:
  - PR #10 mtbc 'Remove Guava dependency and improve error for large planes.'
    - src/main/java/omeis/providers/re/RGBIntBuffer.java

--conflicts

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-build-push#29](https://web-proxy.openmicroscopy.org/west-ci/job/OMERO-build-push/29/). See the [console output](https://web-proxy.openmicroscopy.org/west-ci/job/OMERO-build-push/29/consoleText) for more details.
Possible conflicts:
  - PR #10 mtbc 'Remove Guava dependency and improve error for large planes.'
    - src/main/java/omeis/providers/re/RGBIntBuffer.java

--conflicts

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-build-push#30](https://web-proxy.openmicroscopy.org/west-ci/job/OMERO-build-push/30/). See the [console output](https://web-proxy.openmicroscopy.org/west-ci/job/OMERO-build-push/30/consoleText) for more details.
Possible conflicts:
  - PR #10 mtbc 'Remove Guava dependency and improve error for large planes.'
    - src/main/java/omeis/providers/re/RGBIntBuffer.java

--conflicts

File: src/main/java/ome/util/checksum/AbstractChecksumProvider.java [hunk 1]
```diff
@@ -33,7 +33,7 @@
  * Abstract skeleton class implementing {@link ChecksumProvider} and providing
  * implementations of the interface methods using a universal checksum class
  * object. Classes extending this class shall pass in a concrete checksum
- * algorithm implementation (a {@link HashFunction} instance) as the constructor
+ * algorithm implementation (a @see HashFunction instance) as the constructor
```
Comment (mtbc) [inline src/main/java/ome/util/checksum/AbstractChecksumProvider.java]: Seems to be a regression.

**Pull request merged**